### PR TITLE
Small bug fix

### PR DESCRIPTION
**Pull request by paveltomin**

### CODE REVIEW

**Comment by tjb-ltk** [comment]
Good find.. the constructor initiated value to 1 and the wrong key name some how prevented it from being set to zero.......  However in the fix, the setApplyDefaultValue should be 1, consistent with reservoir default .  I would assume that this value should always be the same as reservoir...

**Comment by paveltomin** [comment]
> Good find.. the constructor initiated value to 1 and the wrong key name some how prevented it from being set to zero....... However in the fix, the setApplyDefaultValue should be 1, consistent with reservoir default . I would assume that this value should always be the same as reservoir...

yes, you are right
just fixed, thanks!

File: src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp [hunk 1]
```diff
@@ -73,8 +73,8 @@ CompositionalMultiphaseWell::CompositionalMultiphaseWell( const string & name,
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Use mass formulation instead of molar" );
 
-  this->registerWrapper( viewKeyStruct::useMassFlagString(), &m_useTotalMassEquation ).
-    setApplyDefaultValue( 0 ).
+  this->registerWrapper( viewKeyStruct::useTotalMassEquationString(), &m_useTotalMassEquation ).
+    setApplyDefaultValue( 1 ).
```
Comment (dkachuma) [inline src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp:77]:
```suggestion
    setApplyDefaultValue( m_useTotalMassEquation ).
```

**Review by dkachuma — APPROVED** [review]
Great find.

**Review by tjb-ltk — APPROVED** [review]

File: src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp (hunk 1 above)
Comment (tjb-ltk) [reply src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp:77]: I think the intent was to use all this  machinery to set the default and avoid initializing value on class construction..

**Comment by paveltomin** [comment]
Rebaseline required due to appearance of `useTotalMassEquation` in well solver params. No real results change.

**Pull request merged**